pkg/crypto/internal/groth16/bls12381/groth16Verify.go

```diff
@@ -0,0 +1,81 @@
+package bls12381
+
+import (
+	"errors"
+	"github.com/consensys/gurvy/bls381/fr"
+	Proof "github.com/wavesplatform/gowaves/pkg/crypto/internal/groth16/bls12381/proof"
```

Comment:
Quote: "When renaming an imported package, the local name should follow the same guidelines as package names (lower case, no under_scores or mixedCaps)" from https://blog.golang.org/package-names.
Why rename if there is no package name collision? If it collides with variable names, like proof in this case, better rename variable to something short, e.g. `proof` to `p`.